server/src/concepts/offer/model/OfferValidation.js

```diff
@@ -0,0 +1,33 @@
+import Joi from "joi-oid";
+
+const validateCreateOffer = (offer) => {
+  const schema = Joi.object({
+    title: Joi.string().min(3).max(50).required(),
+
+    description: Joi.string().max(500).required(),
+
+    city: Joi.string().max(58).required(),
+
+    phoneNumber: Joi.number().min(9).max(9).required(),
+
+    photos: [
```

Comment:
It does not work like that, it has to be something like this:
```
Joi.array().items(Joi.object({
    url: Joi.string()
    isMainPhoto: Joi.boolean()
}))
```
docs: https://joi.dev/api/?v=17.6.0#array